src/Api/Checkout.php

```diff
@@ -0,0 +1,152 @@
+<?php
+
+namespace zaporylie\Vipps\Api;
+
+use zaporylie\Vipps\Model\Checkout\Logistics;
+use zaporylie\Vipps\Model\Checkout\MerchantInfo;
+use zaporylie\Vipps\Model\Checkout\PaymentTransaction;
+use zaporylie\Vipps\Model\Checkout\PrefillCustomer;
+use zaporylie\Vipps\Model\Checkout\RequestAmount;
+use zaporylie\Vipps\Model\Checkout\RequestInitiateSession;
+use zaporylie\Vipps\Model\Checkout\ResponseCancelSession;
+use zaporylie\Vipps\Model\Checkout\ResponseGetSessionDetails;
+use zaporylie\Vipps\Model\Checkout\RequestCancelSession;
+use zaporylie\Vipps\Model\Checkout\ResponseInitiateSession;
+use zaporylie\Vipps\Resource\Checkout\CancelSession;
+use zaporylie\Vipps\Resource\Checkout\GetSessionDetails;
+use zaporylie\Vipps\Resource\Checkout\InitiateSession;
+
+/**
+ * Class Checkout.
+ *
+ * @package Vipps\Api
+ */
+class Checkout extends ApiBase implements CheckoutInterface {
+
+    /**
+     * {@inheritdoc}
+     */
+    public function initiateSession(
+      string $client_secret,
+      string $callback_prefix,
+      string $return_url,
+      string $callback_auth_token,
+      RequestAmount $amount,
+      array $options = [],
+      bool $contact_fields = TRUE,
+      bool $address_fields = TRUE
```

Comment:
Since contact_fields and address_fields are optional values, support them via $options array. You can set to TRUE by default, that's fine, if options arent provided but if options are provided - use that value. Ex.
```
$options = [
  'address_fields' => FALSE,
]
```

Reply:
Done.